langchain4j/src/main/java/dev/langchain4j/model/output/EnumOutputParser.java

```diff
@@ -1,10 +1,16 @@
 package dev.langchain4j.model.output;
 
-import java.util.Arrays;
+import dev.langchain4j.model.output.structured.Description;
+
+import java.lang.reflect.Field;
+import java.util.Locale;
+import java.util.Optional;
 
 @SuppressWarnings("rawtypes")
 public class EnumOutputParser implements OutputParser<Enum> {
 
+    private static final boolean PRETTY = false;
```

Comment:
is this required?

Reply:
It's indeed nood needed... Obsolete artifact I've used for experimenting.